demos/python_demos/crnn_text_recognition_demo/crnn_text_recognition_demo.py

```diff
@@ -0,0 +1,124 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+
+from openvino.inference_engine import IENetwork, IECore
+import cv2
+import numpy as np
+import sys
+import os
+import logging as log
+from argparse import ArgumentParser, SUPPRESS
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+                      help='Show this help message and exit.')
+    args.add_argument('-i',
+                      dest='input_source',
+                      help='Required. Path to the input image.',
+                      required=True, type=str, metavar='"<path>"')
+    args.add_argument('-m', '--model',
+                      help='Required. Path to an .xml file with a trained model.',
+                      required=True, type=str, metavar='"<path>"')
+    args.add_argument('-d', '--device',
+                      help='Optional. Specify the target device to infer on: CPU, GPU, FPGA, HDDL or MYRIAD. '
+                           'The demo will look for a suitable plugin for device specified '
+                           '(by default, it is CPU).',
+                           default='CPU', type=str, metavar='"<device>"')
+    args.add_argument('-l', '--cpu_extension',
+                      help='Required for CPU custom layers. '
+                           'Absolute path to a shared library with the kernels implementation.',
+                      default=None, type=str, metavar='"<absolute_path>"')
```

Comment:
Why is it `"<absolute_path>" ` for `-l` in `metavar`, while for others it's just `"<path>"`?

Reply:
`metavar`'s value should also be updated.